components/sidebar/index.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators, Dispatch} from 'redux';
+
+import {GlobalState} from 'mattermost-redux/types/store';
+import {GenericAction} from 'mattermost-redux/types/actions';
+
+import Sidebar from './sidebar';
+
+function mapStateToProps(state: GlobalState) {
+    return {
+    };
+}
+
+function mapDispatchToProps(dispatch: Dispatch<GenericAction>) {
+    return {
+        actions: bindActionCreators({
+        }, dispatch),
+    };
+}
+
+export default connect(mapStateToProps, mapDispatchToProps)(Sidebar);
```

Comment:
Maybe just replace this whole file with
```
import Sidebar from './sidebar';

export default Sidebar;
```
to avoid adding unneeded code? 0/5

Reply:
Ehh, there's a pretty good chance we're going to need it, so i'll leave it for now.